src/content/blog/astro-470.mdx

```diff
@@ -0,0 +1,71 @@
+---
+title: "Astro 4.7"
+description: "Astro 4.7 is out now! This release includes major improvements to the API for making toolbar apps, a new way to keep yourself up to date, and more."
+publishDate: "April 25, 2024"
+authors:
+  - erika
+  - ema
+  - matthew
+  - nate
+  - bjorn
+  - sarah
+coverImage: "/src/content/blog/_images/astro-470/post-header-4.7.webp"
+socialImage: "/src/content/blog/_images/astro-470/og-image-4.7.webp"
+lang: "en"
+---
+
+import BlogSummary from "/src/components/BlogSummary.astro";
+import updateCheckerImage from "/src/content/blog/_images/astro-470/update-checker.webp";
+import BlogContentImage from "/src/components/BlogContentImage.astro"
+
+<BlogSummary>
+  Astro 4.7 is now available! This release includes major improvements to the API for making toolbar apps, more ways to keep your Astro project up to date, and more.
+</BlogSummary>
+
+Full release highlights include:
+
+- [**Major improvements to the Dev Toolbar API**](#dev-toolbar-api-improvements)
```

Comment:
```suggestion
description: "Astro 4.7 is out now! This release includes astronomical improvements to the API for making toolbar apps, a new way to keep yourself up to date, and more."
publishDate: "April 25, 2024"
authors:
  - erika
  - ema
  - matthew
  - nate
  - bjorn
  - sarah
coverImage: "/src/content/blog/_images/astro-470/post-header-4.7.webp"
socialImage: "/src/content/blog/_images/astro-470/og-image-4.7.webp"
lang: "en"
---

import BlogSummary from "/src/components/BlogSummary.astro";
import updateCheckerImage from "/src/content/blog/_images/astro-470/update-checker.webp";
import BlogContentImage from "/src/components/BlogContentImage.astro"

<BlogSummary>
  Astro 4.7 is now available! This release includes *astronomical* improvements to the API for making toolbar apps, more ways to keep your Astro project up to date, and more.
</BlogSummary>

Full release highlights include:

- [**Out-of-this-world improvements to the Dev Toolbar API**](#dev-toolbar-api-improvements)
```
Was trying to think of an alternative to major, so here's one big chunk where the word occurs. Do you want to go with some combination of "astronomical/out-of-this-world"? 😂 

Down in the section itself, I used "significant", which could also be repeated. Words like "extensive" or "considerable" are other neutral choices. Your pick! Maybe you're even fine saying "major"!

Reply:
I'm a bit boring! I like significant. I think I'll use significant in the title, and considerable in the text.